source/src/BNFC/Backend/CPP/STL/CFtoBisonSTL.hs

```diff
@@ -124,8 +125,7 @@ header inPackage name cf = unlines
     , "void " ++ ns ++ "yyerror(const char *str)"
     , "{"
     , "  extern char *"++ns++"yytext;"
```

Comment:
The declaration of `yytext` is now redundant and triggers a compiler warning on GNU C++ (not clang however):
```
test.y: In function 'void yyerror(const char*)':
test.y:25:16: warning: unused variable 'yytext' [-Wunused-variable]
   25 |   extern char *yytext;
      |                ^~~~~~
```

Reply:
Fixed by 3b751a6dad594c6734c1474561a7db77ddd6af73.